services/dbt/models/datenspende/user_research/vitaldata_per_day.sql

```diff
@@ -0,0 +1,14 @@
+{{ config(materialized='table') }}
+
+with vitals_per_day as (
```

Comment:
Why the wrapping in `with ... as` and `select * from ...`

Reply:
dbt style/syntax based off their tutorial